aws-codeartifact-repository/src/main/java/software/amazon/codeartifact/repository/Translator.java

```diff
@@ -48,11 +61,44 @@ static CreateRepositoryRequest translateToCreateRequest(final ResourceModel mode
     return CreateRepositoryRequest.builder()
         .domain(model.getDomainName())
         .domainOwner(model.getDomainOwner())
+        .upstreams(translateToUpstreamList(model))
         .repository(model.getRepositoryName())
         .description(model.getDescription())
         .build();
   }
 
+  /**
+   * Translates ResourceModel upstream array into UpstreamRepository List
+   * @param model resource model
+   * @return list of UpstreamRepository
+   */
+  static List<UpstreamRepository> translateToUpstreamList(final ResourceModel model) {
```

Comment:
I'm curious about naming conventions. If it were up to me I'd drop 'translate' from the method name because both the class name when calling the static method and the verb `to` make it clear that this is a conversion. But I don't know if this Translator class is a standard convention for CFN and if the `translateTo` method name is part of that convention.

Reply:
The naming here was auto-generate thru a boostrapt script. I do agree its a but redundant, I'll leave it as is since that's what the convention seems to be